aiokafka/consumer/group_coordinator.py

```diff
@@ -393,15 +400,23 @@ async def _on_join_prepare(self, previous_assignment):
                               " for group %s failed on_partitions_revoked",
                               self._subscription.listener, self.group_id)
 
-    async def _perform_assignment(
-        self, leader_id, assignment_strategy, members
-    ):
+    async def _perform_assignment(self, response: Response):
+        assignment_strategy = response.group_protocol
+        members = response.members
         assignor = self._lookup_assignor(assignment_strategy)
         assert assignor, \
             'Invalid assignment protocol: %s' % assignment_strategy
         member_metadata = {}
         all_subscribed_topics = set()
-        for member_id, metadata_bytes in members:
+        for member in members:
+            if isinstance(response, JoinGroupResponse[3]):
```

Comment:
_nit:_ It's not obvious that `JoinGroupResponse[3]` is v5, maybe it would be better to use `JoinGroupResponse_vN` explicitly?

Reply:
updated